src/tchannel_bridge.js

```diff
@@ -61,7 +62,8 @@ export default class TChannelBridge {
      * a the handler's context with a span.
      **/
     tracedHandler(handlerFunc: any, options: startSpanArgs = {}): Function {
-        return (context, request, headers, body, callback) => {
+        return (perProcessOptions, request, headers, body, callback) => {
+            let context: DefaultContext = new DefaultContext();
```

Comment:
if you do this, you cannot allow the user to substitute this for RPCContext. My recommendation is to accept a factory method `makeContext = func() { return new DefaultContext(); }` in the constructor and use it to instantiate a context when we need it. It's the same default behavior, but users can override that function with `return new RPCContext()`